chaos-common/src/main/java/io/openchaos/common/InvokeResult.java

```diff
@@ -16,6 +16,7 @@
 public enum InvokeResult {
     SUCCESS,
     FAILURE,
+    TimeOut,
```

Comment:
Actually, the original meaning of UNKOWN is timeout.  You can change the expression of unknown to TIMEOUT. Note that the letters should be capitalized.

Reply:
ok

Reply:
Kafka has used InvokeChaosClient, so I will change timeout to unknown.